packages/host/app/components/operator-mode/card-url-bar.gts

```diff
@@ -38,11 +37,17 @@ export default class CardURLBar extends Component<Signature> {
       ...attributes
     >
       <div class='realm-info' data-test-card-url-bar-realm-info>
-        <div class='realm-icon'>
-          <img src={{this.realmIcon}} alt='realm-icon' />
-        </div>
-        <span>in
-          {{if this.realmName this.realmName 'Unknown Workspace'}}</span>
+        <RealmInfoProvider
+          @realmURL={{this.realmURL}}
+          @returnUnknownRealmIfError={{true}}
```

Comment:
You can use the `error` block in `RealmInfoProvider` to provide custom error displays.

I think displaying "Unknown realm" could be confusing to the user. If there is an error while fetching the realm isn't it better to say exactly that?